pkg/provider/github/github.go

```diff
@@ -64,7 +64,7 @@ type skippedRun struct {
 func New() *Provider {
 	return &Provider{
 		APIURL:        github.String(keys.PublicGithubAPIURL),
-		paginedNumber: defaultPaginedNumber,
+		PaginedNumber: defaultPaginedNumber,
```

Comment:
does it actually need to be initialized from New(), is this used anywhere ?
we can just detect if paginedNumber is 0 and set it to default from inside the call?

It's perhaps only needed for unittests?

it's not really important but can you try to find out if we can refactor this out...

Reply:
we can set the default to 100 without having this as public interface i think...